src/mongocrypt-key.c

```diff
@@ -23,15 +23,19 @@ static bool
 _one_key_alt_name_equal (_mongocrypt_key_alt_name_t *ptr_a,
                          _mongocrypt_key_alt_name_t *ptr_b)
 {
-   BSON_ASSERT (ptr_a->value.value_type == BSON_TYPE_UTF8);
-   BSON_ASSERT (ptr_b->value.value_type == BSON_TYPE_UTF8);
+   BSON_ASSERT (ptr_a && ptr_a->value.value_type == BSON_TYPE_UTF8);
+   BSON_ASSERT (ptr_b && ptr_b->value.value_type == BSON_TYPE_UTF8);
```

Comment:
Suggest asserting not-null separately with `BSON_ASSERT_PARAM`

Reply:
:+1: Change coming in next commit.